scripts/setup_webserver.sh

```diff
@@ -377,7 +381,7 @@ pm.max_spare_servers = 30
 EOF
 
      # Restart fpm
-     service php7.0-fpm restart
+     service php${PhpVer}.0-fpm restart
```

Comment:
Why is `.0` left over here? If `${PhpVer}` is `7.0`, then this line will become `service php7.0.0-fpm restart`, which I don't think will work...?